### PR TITLE
ISPN-16235: Doc edits for topology set-stable command

### PR DESCRIPTION
**Pull request by theashiot**
https://issues.redhat.com/browse/ISPN-16235

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+
+. Set the current topology as stable for the given cache.
 +
 [source,options="nowrap",subs=attributes+]
 ----
 topology set-stable cacheName
 ----
 +
-* If the current topology is missing more nodes than the number of owners, the force flag is necessary to confirm the operation.
+If the current topology is missing more nodes than the number of owners, the force flag is necessary to confirm the operation.
```
Comment (wfink) [inline documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc:18]: it's not 'more nodes than..' but 'equal or more'.

**Review by wfink — CHANGES_REQUESTED** [review]
I would like to have a warning that the 'force' flag need consideration.
If it is necessary, or used by default, it means that some entries are lost.
If NOT all caches have the same numOwner it means that only some caches are affected, so the user might check only those.

File: documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc (hunk 1 above)
Comment (theashiot) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc:18]: updated

**Comment by theashiot** [comment]
Thanks, @wfink for the review! I've made the suggested updates, can you please have a look?

best,
ashwin

File: documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc [hunk 2]
```diff
@@ -16,16 +22,16 @@ However, we offer a CLI command to mark the current topology stable for a specif
 topology set-stable cacheName
 ----
 +
-* If the current topology is missing more nodes than the number of owners, the force flag is necessary to confirm the operation.
+* If the number nodes missing from the current topology is more than or equal to the number of owners, the force flag is necessary to confirm the operation.
```
[the remark this answers is not in the document]
Comment (theashiot) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: Thanks, @tristantarrant for catching this, fixed!

[the remark this answers is not in the document]
Comment (wfink) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: as said in the review comment I would add a warning that this means the cache will loose data.

[the remark this answers is not in the document]
Comment (theashiot) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: Hi @wfink, do you mean https://github.com/infinispan/infinispan/pull/12573/files#diff-ff9447088ee5f555abd6849369c66c62167bed44bacb067efb2a2d7c96fff5b0R32 should be [[WARNING]]? Or should "If you use the force flag, you might lose cache entries." should be reworded as "If you use the force flag, you might lose cache entries and data."

thanks,
ashwin

[the remark this answers is not in the document]
Comment (wfink) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: yes,  sorry missed that

File: documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc [hunk 3]
```diff
@@ -16,16 +22,16 @@ However, we offer a CLI command to mark the current topology stable for a specif
...
 topology set-stable cacheName -f
 ----
-
-For more information about the [command]`topology set-stable` command, run [command]`topology set-stable -h`.
-
++
 [IMPORTANT]
 ====
-Manually installing a topology can lead to data loss, only perform this operation if the initial topology cannot be recreated.
+If you use the force flag, you might lose cache entries. If all the caches do not have the same number of owners, `numOwners`, then only some caches are affected. You can examine the only the caches with a different number of owners.
```
Comment (jabolina) [inline documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc:34]: It seems to mix a couple of concepts. I agree with the force flag part. I didn't get the second part, though. Can you elaborate more?

Also, one thing worth adding is the stating the command applies to a single command. The user would need to execute the command for each cache they want to set the topology.

Comment (theashiot) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: Thanks, @jabolina, updated, wdyt?

**Pull request merged**